### PR TITLE
View Factor Labels Empty Dialog

### PR DESCRIPTION
**Pull request by stevenndungu**
@africanmathsinitiative/developers this is ready for review

### CODE REVIEW

**Comment by stevenndungu** [comment]
@AlexSananka  review this, need to work on it

File: instat/dlgFactorLabels.vb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    Private Sub InitialiseDialog()
+
+    End Sub
+
+    Private Sub SetDefaults()
+
+    End Sub
+
+
```
Comment (AlexSananka) [inline instat/dlgFactorLabels.vb]: One minor and tiny edit,There is a lot of spacing between this subs,Otherwise it's ready for @dannyparsons  to merge

Comment (stevenndungu) [inline instat/dlgFactorLabels.vb]: Intresting, space deleted.

**Comment by AlexSananka** [comment]
@dannyparsons This is ready to be merged.

**Pull request merged**